org/cloudfoundry.yml

```diff
@@ -5944,6 +5951,7 @@ orgs:
           csb-brokerpak-azure: admin
           csb-brokerpak-gcp: admin
           upgrade-all-services-cli-plugin: admin
+          terraform-provider-csbpg: admin
```

Comment:
Should not be needed anymore. The automation will add this repo to https://github.com/orgs/cloudfoundry/teams/wg-service-management-cloud-service-broker-approvers

The non-standard teams are scheduled for removal: #262

Reply:
Thank you @stephanme. I've now removed that line.